lib/wallet/api/http/Cardano/Wallet/Api/Http/Shelley/Server.hs

```diff
@@ -3520,7 +3520,7 @@ balanceTransaction
         let utxoIndex =
                 Write.constructUTxOIndex $
                 Write.fromWalletUTxO utxo
-        partialTx <- parsePartialTx era
+        partialTx <- Write.withRecentEra era parsePartialTx
```

Comment:
```suggestion
        (partialTx :: Write.PartialTx era) <- parsePartialTx
```

where the type `era` needs to brought into scope when pattern matching `PParamsInAnyRecentEra`.

Reply:
Fixed by https://github.com/cardano-foundation/cardano-wallet/compare/d07d079fe50f9ba3ed95fceb14e02c97e9033ad4..642df6baeeaa123bfca4dd48c14e81b58c2278a1.